community/shadow/APKBUILD

```diff
@@ -31,6 +33,7 @@ builddir="$srcdir/shadow-$pkgver"
 
 build() {
 	cd "$builddir"
+	[ ${CBUILD} != ${CHOST} ] && extra_conf="--disable-subordinate-ids"
```

Comment:
The same as above about quoting and condition.

Variable `extra_conf` will be undefined, or defined outside the script with some arbitrary content (!), when the condition is evaluated as false. That’s quite error prone. Moreover, you unnecessarily pollute global space of variables.

Please always declare variables used only in a single function as `local`, i.e. `local extra_conf` or `local extra_conf=""`.